sg/datatablesbundle/1.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Sg\\DatatablesBundle\\SgDatatablesBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "make cache-warmup": "script",
+        "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
+    }
```

Comment:
Looks like this is a copy/paste remaining from framework-bundle, which should be removed.

Reply:
done